src/Simulation/Simulators.Tests/Circuits/GetAvailableTest.qs

```diff
@@ -9,22 +9,22 @@ namespace Microsoft.Quantum.Simulation.Simulators.Tests.Circuits {
         mutable innerBorrow = 0;
         borrowing (qb = Qubit())
         {
-            set innerBorrow = GetQubitsAvailableToBorrow();
+            set innerBorrow = GetQubitsAvailableToBorrow() + GetQubitsAvailableToUse();
         }
-        return (GetQubitsAvailableToBorrow(), innerBorrow);
+        return (GetQubitsAvailableToBorrow() + GetQubitsAvailableToUse(), innerBorrow);
```

Comment:
Are these changes that add together the available to use and available to borrow counts because the contract on the `GetQubitsAvailableToBorrow` method changed or because the way the test executes changed?

Reply:
I changed the interpretation of GetQubitsAvailableToBorrow (and the doc comment on it as well). Before by definition it returned not just all qubits that you could borrow, but also all qubits that weren't allocated/used yet. Since one interpretation can be converted to the other via  + or - GetQubitsAvailabeToUse() the change is certainly not necessary. I merely took the opportunity to do this switch as part of revising the Q# core library in general. @cgranade, @alan-geller for input on which one is better. All uses that I could come up with would require knowing how many qubits can be borrowed and not how many there are, but we should certainly be mindful to not change things unless there is a good reason to and maybe that is not a good enough reason.

Reply:
I think either way makes sense; as you note, it's really easy to convert between. I could see, for instance, a pure-Q# library operation `GetQubitsAvailableToBorrowOrUse` to make the distinction clear.

That said, this is a breaking change that we can't make a deprecation stub for (the operation didn't change names, to wit). I strongly suspect these two operations are rarely used enough that we should be OK, but it would take a bump in minor version and a release note line item.

Reply:
I'll keep a list of breaking changes.